java/src/gov/usgs/processingformats/Site.java

```diff
@@ -220,6 +295,32 @@ public boolean isValid() {
 			errorList.add("Empty Network in Site Class.");
 		}
 
+		// latitude
+		if (jsonLatitude == null) {
+			// latitude not found
+			errorList.add("No Latitude in Site Class.");
+		} else if ((jsonLatitude < -90) || (jsonLatitude > 90)) {
+			// invalid latitude
+			errorList.add(
+					"Latitude in Site Class not in the range of -90 to 90.");
+		}
+
+		// longitude
+		if (jsonLongitude == null) {
+			// longitude not found
+			errorList.add("No Longitude in Site Class.");
+		} else if ((jsonLongitude < -180) || (jsonLongitude > 180)) {
```

Comment:
comparing ints with doubles again, i think the compiler will convert 180 to 180.0, but reading the code is misleading, and if you put what you want, then you know what is getting compared...